eq-author/src/App/questionPage/Preview/index.js

```diff
@@ -80,6 +119,24 @@ export const UnwrappedPreviewPageRoute = ({ loading, data }) => {
             <Panel dangerouslySetInnerHTML={{ __html: guidance }} />
           </Guidance>
         )}
+
+        {(!isEmpty(definitionLabel) || !isEmpty(definitionContent)) && (
```

Comment:
I think this is the same - which I find easier to read.
```suggestion
        {!(isEmpty(definitionLabel) && isEmpty(definitionContent)) && (
```

Reply:
Not too sure I agree.  It's better to have smaller blocks of logic which can be read without having to jump back.  Either way is fine with me 👍

Reply:
Yeah, I agree. I wasn't really sure. Maybe there would be some benefit in extracting this out to a separate function as it is starting to get a little large. I mean rendering the definition preview - not just the condition.

Reply:
However, its just at the limit from my point of view where inline is ok.